README.md

```diff
@@ -21,6 +21,40 @@ To build the production version of this package, simply run `npm install && npm
 Import this into your package as is described in the GitHub examples.
 
 ## Customization
+### Logger Configuration
+The logger is provided by ChatJS package, you can configure it in this file: `src/utils/log.js`.
```

Comment:
nit: ChatJS -> amazon-connect-chatjs, can you also add a link here 
https://github.com/amazon-connect/amazon-connect-chatjs)[amazon-connect-chatjs](https://github.com/amazon-connect/amazon-connect-chatjs